tests/MongoDB.Driver.Tests/Specifications/crud/CrudOperationTestFactory.cs

```diff
@@ -0,0 +1,45 @@
+﻿/* Copyright 2018-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+
+namespace MongoDB.Driver.Tests.Specifications.crud
+{
+    public static class CrudOperationTestFactory
```

Comment:
I factored the creation of the CrudOperation JSON tests into a factory so that the factory could shared with the change stream JSON test runner.